packages/devtools-reps/src/object-inspector/tests/component/classnames.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { mount } = require("enzyme");
+const React = require("react");
+const { createFactory } = React;
+const ObjectInspector = createFactory(require("../../index"));
+
+function generateDefaults(overrides) {
+  return Object.assign({
+    autoExpandDepth: 0,
+    roots: [{
+      path: "root",
+      name: "root",
+      contents: {value: 42}
+    }],
+    getObjectProperties: () => {},
+    loadObjectProperties: () => {},
+  }, overrides);
+}
+
+describe("ObjectInspector - classnames", () => {
+  it("has the expected class", () => {
+    const props = generateDefaults();
+    const oi = ObjectInspector(props);
+    const wrapper = mount(oi);
+
+    expect(wrapper.hasClass("tree")).toBeTruthy();
+    expect(wrapper.hasClass("inline")).toBeFalsy();
+    expect(wrapper.hasClass("nowrap")).toBeFalsy();
+    expect(oi).toMatchSnapshot();
```

Comment:
you might want to try `expect(oi.debug()).toMatchSnapshot();`

this will give you the html, which will be more releated